Sources/Turf/WKT.swift

```diff
@@ -0,0 +1,151 @@
+import Foundation
+
+/**
+ Entity which can be converted to and from 'Well Known Text'.
+ */
+public protocol WKTConvertible {
+    var wktString: String { get }
+    init(fromWKT wkt: String) throws
```

Comment:
The “from” in the argument label is unnecessary. There does still need to be a `wkt` label because not every string is a WKT string:

```swift
init(wkt: String) throws
```